<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MBI_GUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>388</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Bucket 1</string>
    </property>
    <widget class="QWidget" name="ImgWidget1" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>368</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="DispImage">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Image</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>388</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Bucket 2</string>
    </property>
    <widget class="QWidget" name="ImgWidget2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>368</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="Zoom1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QSlider" name="Zoom2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>460</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>460</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QSlider" name="X1">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>390</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>184</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Y1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>22</width>
      <height>301</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>160</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="Y2">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>70</y>
      <width>22</width>
      <height>301</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>160</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="X2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>390</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>184</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="X1box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="X2box">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>390</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Y1box">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Y2box">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>40</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bitLoad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Bit File</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DispType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>ALL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CEP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CEP-TOF</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="maskslabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Masks</string>
    </property>
   </widget>
   <widget class="QLabel" name="maskschangeslabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mask Changes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubChange">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="exposurelabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exposure Time [us]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Masks">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>600</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaskChanges">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Exposure">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maskschangeslabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Subscenes per Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveImages">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Images</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Images</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumImages">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="RecVideo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Record Video</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
